contracts/linear/src/epoch_actions.rs

```diff
@@ -319,6 +340,21 @@ impl LiquidStakingContract {
                 amount: &U128(amount),
             }
             .emit();
+
+            validator
+                .sync_account_balance()
+                .then(ext_self_action_cb::validator_get_account_callback(
```

Comment:
What I think is adding this return in `validator_get_account_callback` is unreasonable because this return is meaningless to the function. So how about using a separate `validator_return_true_callback` to make it clear?